.github/main.workflow

```diff
@@ -32,7 +32,7 @@ action "Pylint" {
   ]
   env = {
     PYTHON_VERSION = "3.7"
-    PIP_PACKAGES = "pylint .[dev]"
+    PIP_PACKAGES = "pylint .[tests]"
```

Comment:
could just add `pylint` to the `tests` dependencies?

Reply:
I'd rather not (as with bandit)